src/box/sql/vdbe.c

```diff
@@ -1976,13 +1976,26 @@ case OP_Column: {
  * P2 fields, then extract a NULL.
  */
 case OP_Fetch: {
-	struct vdbe_field_ref *field_ref =
-		(struct vdbe_field_ref *) p->aMem[pOp->p1].u.p;
-	uint32_t field_idx = pOp->p2;
-	struct Mem *dest_mem = vdbe_prepare_null_out(p, pOp->p3);
-	if (vdbe_field_ref_fetch(field_ref, field_idx, dest_mem) != 0)
+	struct vdbe_field_ref *ref = p->aMem[pOp->p1].u.p;
+	struct tuple_format *format = ref->format;
+	size_t id;
+	if (pOp->p4type != P4_DYNAMIC) {
```

Comment:
You might want to update `OP_Fetch` comment about the new behaviour.

Reply:
Fixed.